GNUmakefile

```diff
@@ -91,12 +91,11 @@ abigen: ## Build & install abigen.
 
 .PHONY: go-solidity-wrappers
 go-solidity-wrappers: pnpmdep abigen ## Recompiles solidity contracts and their go wrappers.
-	./contracts/scripts/native_solc_compile_all
```

Comment:
go generate ./core/gethwrappers already calls compile_all, this should speed up wrapper generation by ~2 minutes